### ISSUE TITLE
[Mission] Ranperre's Final Rest

### ISSUE THREAD

**Issue by Ninjistix** [DarkstarProject/darkstar#5445]
**_I have:_**

- [X] searched existing issues (http://github.com/darkstarproject/darkstar/issues/) to see if the issue I am posting has already been addressed or opened by another contributor
- [X] checked the commit log to see if my issue has been resolved since my server was last updated

**_Client Version_** (type `/ver` in game) **:** 
30181103_0

**_Source Branch_** (master/stable) **:** 
master

**_Additional Information_** (Steps to reproduce/Expected behavior) **:** 

San d'Oria Mission 6-2

The steps for 

* Return to Prince Trion who'll instruct you to go see the Gate Guard. The Gate Guard will tell you that it will take some time to decipher. 
-does not work, visiting the prince room does nothing, but talking to the guard twice will make him accept the KI.

* Zone and talk to the guard again who will tell you to see Prince Trion.
-does not work, I checked the code and it has a wait clause of 1 gameday before the guard lets you progress

* Speak with Prince Trion again.
-unable to verify still waiting on previous step
* Return to the Heavy Stone Door again for another cutscene.
-unable to verify still waiting on previous step
* Return to a Gate Guard to complete the mission.
-unable to verify still waiting on previous step

**Comment by zynjec** [comment]
The gate guard is currently doing a real life day wait (which should have been a game day wait to begni with), but not it seems to be instant/requires zoning.

**Comment by Ninjistix** [comment]
yeah according to https://www.bg-wiki.com/bg/San_d%27Oria_Mission_6-2

and a few other guides I found for the mission, it should be just a zone and not the wait.

**Comment by tagban** [comment]
Ran into this before. Can be pretty easily adjusted iirc.
